websockets/blob.go

```diff
@@ -0,0 +1,75 @@
+package websockets
+
+import (
+	"bytes"
+
+	"github.com/grafana/sobek"
+
+	"go.k6.io/k6/js/common"
+)
+
+type Blob struct {
+	typ  string
+	data bytes.Buffer
+}
+
+func (b *Blob) text() string {
+	return b.data.String()
+}
+
+func (r *WebSocketsAPI) blob(call sobek.ConstructorCall) *sobek.Object {
+	var blobParts []interface{}
+	if len(call.Arguments) > 0 {
+		if parts, ok := call.Arguments[0].Export().([]interface{}); ok {
+			blobParts = parts
+		}
+	}
+
+	return newBlob(r.vu.Runtime(), blobParts)
+}
```

Comment:
That's also another open question: do we really want the `Blob` constructor to be available widely? Otherwise I guess we can just remove this.

Reply:
I don't see a reason why not to and given that people will likely also want to be able to use at as a thing to *send* over the websocket it seems like a requirement.

Reply:
Super! 🎉